src/compatTest/groovy/org/ajoberstar/gradle/git/publish/BaseCompatTest.groovy

```diff
@@ -360,6 +360,29 @@ gitPublish {
     remote.head().fullMessage == "Deploy docs to gh-pages (${projectFile('.').canonicalFile.name})"
   }
 
+  def 'can deactivate signing'() {
```

Comment:
This test only seems to cover that the field can be set. Might be good to confirm it actually prevents signing. The equivalent test in Grgit might be a place to start.

Reply:
Added some tests. I copied the bits from the grgit project for that. You might want to think about using [Gradles test fixtures feature](https://docs.gradle.org/current/userguide/java_testing.html#sec:java_test_fixtures) to share the code.